package.json

```diff
@@ -24,6 +24,9 @@
     "@bazel/jasmine": "1.1.0",
     "@bazel/typescript": "1.1.0",
     "@types/jasmine": "3.5.0",
+    "@types/shelljs": "^0.8.6",
+    "@types/mock-fs": "^4.10.0",
+    "@bazel/bazel": "^2.0.0",
```

Comment:
Can you remove the range selectors on the version number? We keep the deps as specific as possible.

Also the addition of the `@bazel/bazel` package seems out of scope here. We should pick a way to obtain bazel and go with it, currently that's bazelisk. If we think the bazel npm package is better suited we can switch.

Reply:
Ah right, we don't use bazelisk so not familar with it.
I think there's a issue in rules_nodejs to adjust @bazel/bazel to use it at some point though.
Anyway, ill remove it for now